Slim/Plugin/Podcast/Provider.pm

```diff
@@ -0,0 +1,23 @@
+package Slim::Plugin::Podcast::Provider;
+
+# Logitech Media Server Copyright 2005-2020 Logitech.
+
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License,
+# version 2.
+
+use strict;
+
+sub new { shift };
+
+sub getMenuItems { [ { } ] }
+
+sub parseStart {
+	return {
+		index => 0,
+		feeds => $_[1],
+	};
+}
```

Comment:
Needs to be replaced with a stub `getFeedsIterator()`.

Reply:
I need to fully remove it as there is no stub anymore